bot/pagination.py

```diff
@@ -62,23 +73,33 @@ def add_line(self, line: str = '', *, empty: bool = False) -> None:
         """
         Adds a line to the current page.
 
-        If the line exceeds the `self.max_size` then an exception is raised.
+        If the line exceeds `self.max_size`, then `self.max_size` will go up to `scale_to_size` for
+        a single line before creating a new page. If it is still exceeded, the excess characters
+        are stored and placed on the next pages until there are none remaining (by word boundary).
```

Comment:
I think this still needs a bit of work. It's not immediately clear to me how this works from this explanation. Furthermore, it fails to communicate _why_ we need two variables to determine the length of this line.

Reply:
Hmm, okay. I'll try to work on expanding this docstring a bit. In hindsight, I agree that it doesn't really address the reasoning behind having both `max_size` and `scale_to_size`.